test/e2e/performanceprofile/functests/1_performance/performance.go

```diff
@@ -290,15 +290,47 @@ var _ = Describe("[rfe_id:27368][performance]", Ordered, func() {
 	})
 
 	Context("RPS configuration", func() {
-		It("[test_id:55012] Should have the correct RPS configuration", func() {
+		BeforeEach(func() {
 			if profile.Spec.CPU == nil || profile.Spec.CPU.Reserved == nil {
 				Skip("Test Skipped due nil Reserved cpus")
 			}
+		})
+		It("[test_id: 59572] Check RPS Mask is applied to atleast one single rx queues of a veth interface", func() {
+			if profile.Spec.WorkloadHints != nil && profile.Spec.WorkloadHints.RealTime != nil &&
+				!*profile.Spec.WorkloadHints.RealTime && !profileutil.IsRpsEnabled(profile) {
+				Skip("realTime Workload Hints is not enabled")
+			}
+			expectedRPSCPUs, err := cpuset.Parse(string(*profile.Spec.CPU.Reserved))
+			Expect(err).ToNot(HaveOccurred())
+			for _, node := range workerRTNodes {
+				//Get one veth interface
+				count := 0
+				cmd := []string{"/bin/sh", "-c", fmt.Sprintf("ip -c link show type veth | head -n 1")}
```

Comment:
this will get the first veth in the kernel order. So no guarantees about which veth will we work with. Is this good enough for this test? if so deserves a comment

Reply:
Maybe checking under `/sys/class/net/` is safer

Reply:
@Tal-or I have used /sys/clas/net in the latest commit. Please check